platform/darwin/test/MGLCoordinateFormatterTests.m

```diff
@@ -24,7 +24,12 @@ - (void)testStrings {
     coordinate = CLLocationCoordinate2DMake(38.9131982, -77.0325453144239);
     XCTAssertEqualObjects([shortFormatter stringFromCoordinate:coordinate], @"38°54′48″N, 77°1′57″W");
     XCTAssertEqualObjects([mediumFormatter stringFromCoordinate:coordinate], @"38°54′48″ north, 77°1′57″ west");
-    XCTAssertEqualObjects([longFormatter stringFromCoordinate:coordinate], @"38 degrees, 54 minutes, and 48 seconds north by 77 degrees, 1 minute, and 57 seconds west");
+    if (@available(iOS 9.0, *)) {
+        XCTAssertEqualObjects([longFormatter stringFromCoordinate:coordinate], @"38 degrees, 54 minutes, and 48 seconds north by 77 degrees, 1 minute, and 57 seconds west");
+    } else {
+        // Foundation in iOS 8 does not know how to pluralize coordinates.
```

Comment:
The stringsdict format was introduced in iOS 7. I wonder if the problem is that Foundation.stringsdict is located in en.lproj whereas Foundation.strings is base-localized.